src/userCLI/scripts/build

```diff
@@ -3,6 +3,5 @@ set -ex
 
 HERE=$(dirname $0)
 . $HERE/common
-(cd ..;
-    ./gradlew :userCLI:distDocker -i -Pdocker_version=$GIT_SHA -Pdocker_name=$TAG_IMAGE;
-    ./gradlew :userCLI:distDocker -i -Pdocker_version=$GIT_BRANCH -Pdocker_name=$TAG_IMAGE;)
+
+$HERE/../../gradlew :userCLI:distDocker -i -Pdocker_version=$GIT_SHA -Pdocker_name=$TAG_IMAGE;
```

Comment:
I wanted to run this from the top level dir, so I made it agnostic about where it's run from